calico-enterprise_versioned_docs/version-3.18-2/release-notes/index.mdx

```diff
@@ -184,4 +184,25 @@ Calico Enterprise 3.18.2 is now available. This release includes bug fixes and i
 
 #### Updating
 
+To update an existing installation of {{prodname}} 3.18, see [Install a patch release](../getting-started/manifest-archive.mdx).
+
+### Calico Enterprise 3.18.3 bug fix update
+
+April DD, 2024
+
+Calico Enterprise 3.18.3 is now available. This release includes bug fixes and improvements.
+
+### Improvements
+
+* one
+* two
+
+
+### Bug fixes
+
+* one
```

Comment:
1. Fixed an issue where the token login page is no longer working when an identity provider is configured.
2. Fix that workload egress traffic could be incorrectly handled as host-sourced traffic when DNS policy was used in conjuction with IPVS kube-proxy. 
3. Security updates.

Reply:
Added.